### PR TITLE
Use the SVG logo hosted by us

### PR DESCRIPTION
**Pull request by mayhem**
Host our own logo. Should I use url_for here as well? Why is .js where is it?

http://i0.kym-cdn.com/photos/images/newsfeed/000/234/765/b7e.jpg

### CODE REVIEW

**Comment by alastair** [comment]
js is not static because it has some parameters (user id, auth token). These are added using flask templating.

File: webserver/static/img/listenbrainz-logo.svg [hunk 1]
```diff
@@ -0,0 +1,548 @@
+
```
Comment (alastair) [inline webserver/static/img/listenbrainz-logo.svg]: I think you committed a github page

Comment (mayhem) [inline webserver/static/img/listenbrainz-logo.svg]: ha!

Would you believe that chrome manage to load that, regardless??

File: webserver/templates/user/scraper.js [hunk 2]
```diff
@@ -174,5 +174,5 @@ function reportPageAndGetNext(response) {
 }
 
 document.body.insertAdjacentHTML( 'afterbegin', '<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">');
-document.body.insertAdjacentHTML( 'afterbegin', '<div style="position:absolute; top:200px; z-index: 200000000000000; width:500px; margin-left:-250px; left:50%; background-color:#fff; box-shadow: 0 19px 38px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22); text-align:center; padding:50px;" id="listen-progress-container"><img src="https://i.imgur.com/7g0M2fW.jpg" height="75"></div>');
+document.body.insertAdjacentHTML( 'afterbegin', '<div style="position:absolute; top:200px; z-index: 200000000000000; width:500px; margin-left:-250px; left:50%; background-color:#fff; box-shadow: 0 19px 38px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22); text-align:center; padding:50px;" id="listen-progress-container"><img src="/static/img/listenbrainz-logo.svg" height="75"></div>');
```
Comment (alastair) [inline webserver/templates/user/scraper.js]: I think you should use an absolute url with domain name, remember that this file executes on last.fm

**Pull request merged**